std/StringTools.hx

```diff
@@ -413,6 +413,15 @@ class StringTools {
 		return false;
 		#end
 	}
+	
+	/**
+		Converts the first character of the String `s` to upper case and the 
+		remaining characters to lower case.
```

Comment:
Mutating all the letters seems to be generally perceived as the default behaviour. 

[Capitalization is writing a word with its first letter as a capital letter (upper-case letter) and the remaining letters in lower case in writing systems with a case distinction.](http://en.wikipedia.org/wiki/Capitalization).

The same functionality can be found in e.g. [Python](https://docs.python.org/2/library/stdtypes.html#str.capitalize).